src/interfaces/WrapperLike.ts

```diff
@@ -0,0 +1,25 @@
+import { DOMWrapper } from '../domWrapper'
+
+export default interface WrapperLike {
+  find<K extends keyof HTMLElementTagNameMap>(
+    selector: K
+  ): DOMWrapper<HTMLElementTagNameMap[K]>
+  find<K extends keyof SVGElementTagNameMap>(
+    selector: K
+  ): DOMWrapper<SVGElementTagNameMap[K]>
+  find<T extends Element>(selector: string): DOMWrapper<T>
+  find(selector: string): DOMWrapper<Element>
```

Comment:
I think there should be the `get` method as well here

Reply:
I'm not sure how to implement that given the Omit on the get see: https://github.com/vuejs/vue-test-utils-next/pull/372#issuecomment-776610237
I could have a second optional type argument but that felt messy.

Reply:
Hmm I think the Omit should be in both signatures, as `get` throws if the element is not there, so calling `exists` on it has no sense. If that's not the case that's a mistake? Or am I missing something @lmiller1990 ?

Reply:
I think this is probably a mistake, yes